app/assets/stylesheets/legislation.scss

```diff
@@ -69,10 +69,14 @@
 }
 
 .legislation-text {
-  margin-bottom: 1rem;
```

Comment:
I made a mistake here. This line should be kept as well; I removed it thinking it applied to the `.legislation-text` element which is removed in this pull request, but it applies to the element which it **not** removed as well.

Thanks @taitus for noticing.

Reply:
Done!